spec/twitter-ads/creative/promoted_tweet_spec.rb

```diff
@@ -23,7 +23,7 @@
   # check model properties
   subject { described_class.new(account) }
   read  = %w(id approval_status created_at updated_at deleted)
-  write = %w(line_item_id tweet_id paused)
```

Comment:
Make sure you're running at least v0.2.4 where we've already fixed this issue.
https://github.com/twitterdev/twitter-ruby-ads-sdk/releases/tag/v0.2.4